app/models/api_token.rb

```diff
@@ -0,0 +1,32 @@
+class APIToken < ApplicationRecord
+  belongs_to :user
+
+  validates :user, presence: true
+  validates :token, presence: { message: I18n.t('activerecord.errors.models.api_token.token.blank') },
+                    uniqueness: { scope: :user_id },
+                    length: { minimum: 36, maximum: 36 }
+  validates :name,  presence: { message: I18n.t('activerecord.errors.models.api_token.name.blank') },
+                    length: { minimum: 2, maximum: 64 },
+                    uniqueness: { scope: :user_id, message: I18n.t('activerecord.errors.models.api_token.name.uniqueness') }
```

Comment:
As above, message shouldn't be needed if the message is in the right place in `en.yml`